scripts/generate-integration-template-zod.bash

```diff
@@ -36,7 +40,10 @@ for d in "${integrations[@]}" ; do
 
     mv "$integration/nango.yaml" .
 
-    npx nango generate && npx ts-to-zod .nango/schema.ts $integration/schema.zod.ts
+    # if no zod file already then create it unless we want to explicitly override it
+    if [ ! -f "$integration/schema.zod.ts" ] || [ -n "$npm_config_force" ]; then
+        npx nango generate && npx ts-to-zod .nango/schema.ts $integration/schema.zod.ts
+    fi
```

Comment:
would it make sense to document this flag in the readme? (and all other ones)

Reply:
Yup absolutely. I also need to streamline the scripts to be a bit more DRY

Reply:
Added with https://github.com/NangoHQ/integration-templates/pull/40/commits/b8edf65a5f63465c8d9f037670ec866eaa78be13